test-lib-openshift.sh

```diff
@@ -112,8 +112,14 @@ function ct_os_run_in_pod() {
 # Arguments: service_name - name of the service
 function ct_os_get_service_ip() {
   local service_name="${1}" ; shift
+  ocp_docker_address='172\.30\.[0-9\.]*'
```

Comment:
I would prefer to set `ocp_docker_address` as a local variable.

Reply:
Make sense

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83